test-i2c/IMU_v2.ino

```diff
@@ -0,0 +1,141 @@
+//Component: Yost 3-Space™ LX Embedded part number TSS-LX
+
+#include "Yost.h"
+#include <Wire.h>
+
+#define TRUE 1
+#define FALSE 0
```

Comment:
I don't think you need to define these in Arduino, I think `true` and `false` are already mapped to `1` and `0`